exercises/colorsquare.js

```diff
@@ -14,8 +14,15 @@ class ColorSquare{
 		you'll need to bind the handleClick method to this object:
 			this.handleClick = this.handleClick.bind(this)
 	*/
-	constructor( ){
+	constructor(colorsArray, colorIndex, domElementClass ){
+	    this.colorsArray = colorsArray;
+	    this.colorIndex = colorIndex;
+	    this.domElementClass = domElementClass;
+	    this.domElement;
```

Comment:
I usually like to set these to null so I know if they somehow accidentally became undefined.